modules/cnf-about-topology-aware-lifecycle-operator-config.adoc

```diff
@@ -0,0 +1,81 @@
+
+`UpgradeNotStarted`:: In this state, the `ClusterGroupUpgrade` CR is created and the `remediationAction` field is set to `inform`.
++
+The operator builds a remediation plan based on the `clusters` list and the `remediationStrategy` fields. The operator transitions to the `UpgradeNotCompleted` state once the `remediationAction` field is set to `enforce`.
++
+.Sample CR
++
+[source,terminal]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: test
+  namespace: default
+spec:
+  clusters:
+  - spoke1
+  - spoke2
+  - spoke3
+  - spoke4
+  - spoke5
+  - spoke6
+  remediationAction: inform
+  remediationStrategy: <1>
+    Canaries: <2>
+    - spoke1
+    maxConcurrency: 2 <3>
+  platformUpgrade:
+    channel: stable-4.8
+    desiredUpdate:
+     version: 4.8.1
+  	image: quay.io/openshift-release-dev/ocp-release:4.8.1-x86_64
+      force: true
+    upstream: https://amd64.ocp.releases.ci.openshift.org/graph
+----
+<1> The admin can make changes to the `clusters` and the `remediationStrategy` fields only in this state.
+<2> If the `canaries` field is defined with a list of clusters, the first batch of the remediation plan contains the canary clusters.
+<3> The number of remediation batches is the number of clusters divided by `maxConcurrency`. In this example, three remediation batches are created with two spoke clusters in each batches.
+
+`UpgradeNotCompleted`:: In this state, the operator enforces the policies following the remediation plan defined in the `UpgradeNotStarted` state.
++
+Enforcing the policies of subsequent batches start as soon as the current batch policies are all compliant. If the batch times out, the operator moves on to the next batch. The value of this batch timeout is the `timeout` field divided by the number of batches in the remediation plan.
++
+The platform update is enforced first within a batch. Once the platform update policy is compliant, the operator update policies are enforced.
++
+The operator transitions to the `UpgradeTimedOut` state in two cases:
+
+* If the first batch of `ClusterGroupUpgrade` is set to `canaries` and the policies for this first batch are not compliant within the batch timeout
+* If the policies for the update do not turned compliant within the `timeout` value specified in the `remediationStrategy`
+
+`UpgradeTimedOut`:: In this state, the operator periodically checks if all the policies for the `ClusterGroupUpgrade` are compliant. If the policies are compliant, the operator transitions to the `UpgradeCompleted` state.
+The periodic checks allow the updates to complete if the updates take longer due to network, CPU, or other issues. 
+
+`UpgradeCompleted`:: In this state, the cluster updates are complete.
++
+If the `deleteObjectsOnCompletion` field is set to true, the operator deletes the underlying {rh-rhacm} policies. With this configuration, the {rh-rhacm} Hub does not check for compliance after a successful update.
```

Comment:
```suggestion
If the `deleteObjectsOnCompletion` field is set to `true`, the Operator deletes the underlying {rh-rhacm} policies. With this configuration, the {rh-rhacm} Hub does not check for compliance after a successful update.
```

Reply:
The `deleteObjectsOnCompletion`  field is not included in the Sample CR. Please include it and define the field in a call out similar to the previous specs .